desafio-7/desafio-7.css

```diff
@@ -0,0 +1,44 @@
+
+body{
+  background-color:#7c8d9b		;
+  color: white;
+  text-align: justify;
+}
+.info-about{
+  text-align: left;
+
+}
+.container-info{
+  text-align: justify;
+}
+.about{
+  text-align: left;
```

Comment:
this style is not being used, because there’s a styled  `.footer-menu p:    text-align: center;`